templates/node/node.py

```diff
@@ -36,46 +87,28 @@ def ovc(self):
         """
         if self._ovc is not None:
             return self._ovc
-        
-        instance = self.vdc.ovc.instance
-        self._ovc = j.clients.openvcloud.get(instance=instance)
+
+        self._ovc = j.clients.openvcloud.get(instance=self.config['ovc'])
 
         return self._ovc
 
     @property
     def vdc(self):
-        if self._vdc:
-            return self._vdc
-
-        # Get object for an VDC service, make sure exactly one is running
-        vdc = self.api.services.find(template_uid=self.VDC_TEMPLATE, name=self.data['vdc'])
-        if len(vdc) != 1:
-            raise RuntimeError('found %s vdc, requires exactly 1' % len(vdc))
-        
-        self._vdc = vdc[0]
-
+        '''
+        vdc service instance
+        '''
+        self.config
         return self._vdc
 
     @property
     def space(self):
-        return self.ovc.space_get(
-            accountName=None,
-            spaceName=self.data['vdc']
-            )
+        account = self.config['account']
+        vdc = self.config['vdc']
 
```

Comment:
for me this is trying to get new OVC connection, not the one created by `openvcloud` service. Shouldn't it be something like:
``` python
self.ovc.space_get(instance=self.data['ovc'])
```